wallet/src/de/schildbach/wallet/ui/ForgotPinActivity.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019 Dash Core Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.schildbach.wallet.ui
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.MenuItem
+import androidx.appcompat.app.AppCompatActivity
+import de.schildbach.wallet_test.R
+import kotlinx.android.synthetic.main.activity_forgot_pin.*
+
+class ForgotPinActivity : AppCompatActivity() {
+
+    companion object {
+
+        @JvmStatic
+        fun createIntent(context: Context): Intent {
+            return Intent(context, ForgotPinActivity::class.java)
+        }
```

Comment:
I don't see a place where this code is called to start this activity.

Reply:
As we discussed in NMA-14 I left this activity for future use, it is not used anywhere currently.